src/frontend/js/modules/header/header.js

```diff
@@ -31,6 +31,8 @@
 
     .constant('ESN_SUBHEADER_HEIGHT_MD', 47)
 
+    .constant('ACCOUNT_APPLICATION_NAME', 'Account')
```

Comment:
If name is only used to find the application on application grid, I think it should be more explicit like `ACCOUNT_GRID_APPLICATION_NAME` of something like this.